src/Microsoft.Health.Core/Features/Events/IEventGridPublisher.cs

```diff
@@ -0,0 +1,39 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Messaging.EventGrid;
+
+namespace Microsoft.Health.Abstractions.Features.Events
```

Comment:
Generally would need to be in the same namespace as the file.

Reply:
My bad. After the file move forgot to change it.